### ISSUE TITLE
Server/SSR Support

### ISSUE THREAD

**Issue by Pikachews** [oklas/react-through#15]
Unfortunately, it seems this amazing library that does almost everything I want it to do does not support running on the server ☹️ Are there any plans to support this?

**Comment by oklas** [comment]
Yes it is #13 already planned, significant differences in react core become it is not just update deps, coming soon

**Comment by oklas** [comment]
Hi @Pikachews, I confused a little what you asked, SSR and support of new react core. The support of new react core fetaures was already done some time before. The question about SSR has a subtlety in case of dinamic content like remote api fetching or db responces and auch this library too. There are tests for all cases. Some things must works immidiately, but some thins requires special approach. It dependent from props too [check this](https://react-through.js.org/api/createadvagent). May be some documentation improvements required. To clarify what is your problem access to your case is required. So simple example which demonstrate a problem is required. Or better you can contact to me directly, check my profile page here

**Comment by Pikachews** [comment]
Essentially, I would like https://github.com/oklas/react-breadcrumbs-dynamic to also work on the server, otherwise the initial render of the breadcrumbs is empty until the JS mounts.

**Comment by oklas** [comment]
Full environment where problem is reproduced is required, (used stack/boilerplate with db/other dynamic data). Not only mention of used lib. Simple example which demonstrate a problem is required.
